RFCs/FS-1112-obsolete-allowed-to-use-obsolete.md

```diff
@@ -0,0 +1,114 @@
+# F# RFC FS-1112 - Obsolete allowed to use Obsolete
+
+<!--The design suggestion [Obsolete allowed to use Obsolete](https://github.com/fsharp/fslang-suggestions/issues/1055) has been marked "approved in principle".
+-->
+
+This RFC covers the detailed proposal for this suggestion.
+
+- [x] [Suggestion](https://github.com/fsharp/fslang-suggestions/issues/1055)
+- [ ] Approved in principle
+- [ ] [Implementation](https://github.com/dotnet/fsharp/pull/FILL-ME-IN)
+- [ ] Design Review Meeting(s) with @dsyme and others invitees
+- [Discussion](https://github.com/fsharp/fslang-design/discussions/FILL-ME-IN)
```

Comment:
Missing discussion

Reply:
I'm not sure I already should start one, since it's not yet approved